.gitignore

```diff
@@ -0,0 +1 @@
+/node_modules
```

Comment:
Should this rather be `node_modules/` or perhaps `./node_modules/`?

Reply:
Oops! Really sorry you're right it should be `node_modules/` 😅

Reply:
Thanks for changing that!